test/spec/modules/openxAnalyticsAdapter_spec.js

```diff
@@ -1026,4 +1044,256 @@ describe('openx analytics adapter', function() {
+      currency: 'USD',
+      timeToRespond: 200,
+      dealId: 'test-closex-deal-id',
+      responseTimestamp: 1586000000040,
+      ts: 'test-closex-ts'
+    };
+
+    const bidTimeoutOpenX = [{
+      adUnitCode: AD_UNIT_CODE,
+      auctionId: 'test-auction-id',
+      bidId: 'test-openx-request-id'
+    }];
+
+    const bidTimeoutCloseX = [{
+      adUnitCode: AD_UNIT_CODE,
+      auctionId: 'test-auction-id',
+      bidId: 'test-closex-request-id'
+    }];
+
+    const openxAdUnitInfo = [{'code': 'test-div-1',
+      'mediaTypes': {'banner': {'sizes': [[300, 250]]}},
+      'bids': [{'bidder': 'openx',
+        'params': {'unit': '540249866', 'delDomain': 'sademo-d.openx.net'}},
+        {'bidder': 'closex',
+          'params': {'unit': '540249866', 'delDomain': 'sademo-d.openx.net'}}],
+      'sizes': [[300, 250]],
+      'transactionId': 'test-transaction-id'}];
+
+    const auctionEnd = {
+      'auctionId': 'test-auction-id',
+      'timestamp': 1586000000000,
+      'auctionEnd': 1586000000100,
+      'auctionStatus': 'completed',
+      'adUnits': openxAdUnitInfo,
+      'adUnitCodes': [
+        'test-div-1'
+      ],
+      'bidderRequests': [bidRequestedOpenX, bidRequestedCloseX],
+      'noBids': [],
+      'bidsReceived': [bidResponseOpenX, bidResponseCloseX],
+      'winningBids': [bidResponseOpenX],
+      'timeout': 300
+    };
+
+    const bidWonOpenX = {
+      requestId: 'test-openx-request-id',
+      adId: 'test-openx-ad-id',
+      adUnitCode: AD_UNIT_CODE,
+      auctionId: 'test-auction-id'
+    };
+
+    const bidWonCloseX = {
+      requestId: 'test-closex-request-id',
+      adId: 'test-closex-ad-id',
+      adUnitCode: AD_UNIT_CODE,
+      auctionId: 'test-auction-id'
+    };
+
+    let highestBid;
+    const onSlotLoadEvent = {
+      eventType: SLOT_LOADED,
+      args: {
+        slot: {
+          getAdUnitPath: () => {
+            return '/12345678/test_ad_unit';
+          },
+          getSlotElementId: () => {
+            return AD_UNIT_CODE;
+          },
+          getTargeting: () => {
+            return highestBid ? [highestBid.adId] : []
+          }
+        }
+      }
+    };
+    function simulateAuction(events) {
+      events.forEach(event => {
+        const [eventType, args] = event;
+        openxAdapter.track({eventType, args});
+        if (eventType === BID_RESPONSE) {
+          highestBid = highestBid || args;
+          if (highestBid.cpm < args.cpm) {
+            highestBid = args;
+          }
+        }
+      });
+      openxAdapter.slotOnLoad(onSlotLoadEvent.args);
+      openxAdapter.track(onSlotLoadEvent);
+    }
+
+    const SLOT_LOAD_WAIT_TIME = 10;
+    let clock;
+
+    beforeEach(function() {
+      sinon.stub(events, 'getEvents').returns([]);
+      clock = sinon.useFakeTimers();
+    });
+
+    afterEach(function() {
+      events.getEvents.restore();
+      clock.restore();
+    });
+
+    describe('when are bidder wins', function () {
```

Comment:
when there are bidder wins

Reply:
Thanks for catching this.  I'll fix this in my next commit.